build/automation/lib/terraform.mk

```diff
@@ -200,6 +207,14 @@ _terraform-delete-state-lock: ### Delete Terraform state lock - mandatory: STACK
 
 # ==============================================================================
 
+_terraform-copy-common: # Copies all common terraform files to the desired stack - Mandatory: STACK - name of stack to copy common terraform file into
+	cp $(TERRAFORM_DIR)/common/c-*.tf $(TERRAFORM_DIR)/$(STACK)
```

Comment:
Could we use `common` instead of `c` in the file naming convention? This could come with the benefit of file names being more readable when copied to a destination.

Reply:
Yeah, we can do that. I will update the code